cardano-cli/src/Cardano/CLI/Byron/Parsers.hs

```diff
@@ -430,6 +432,12 @@ parseByronVote =
 -- CLI Parsers
 --------------------------------------------------------------------------------
 
+pIsValid :: Parser IsValid
+pIsValid = fmap (IsValid . not) . switch $ mconcat
```

Comment:
Why not use `flag` that defaults to `True`? And then just `fmap` `IsValid`?

See slack conversation for bike shedding of names :smile:

Reply:
The ADT is now called `ScriptValidity`. The option is `--script-valid` and `--script-invalid`